app/main/util/filter_sort_builder.py

```diff
@@ -38,6 +38,23 @@ def build_sorting_orders(args):
     return sorting_orders
 
 
+def build_sorting_orders_open_search(args):
```

Comment:
is this functioning then?

Reply:
the function itself and the query works, the terms at times differ (e.g. it sometimes needs to be foobar.keyword, sometimes metadata.date because its nested in metadata), which makes it slightly more complicated after that point and I'd like to make it slightly cleaner, there is also some logic for passing the template the current filter value thats broken